lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -52,6 +53,8 @@ Creates object instance of the class
 
 =item * C<timeout> - Request timeout, in seconds. If not set, uses the environment variable C<RPC_QUEUE_RESPONSE_TIMEOUT>, or defaults to 300
 
+=item * C<category_timeout_config> - Category timeouts configurations.
```

Comment:
Maybe reword - does not add any new information

Reply:
yeah better to indicates the default values or also an example of the YAML file structure will be good